docs/developer/stock-allocation.mdx

```diff
@@ -43,13 +45,32 @@ classDiagram
     class Warehouse{
       click_and_collect_option: String
     }
+    class Channel{
+      allocationStrategy: String
+    }
 ```
 
+### Allocation strategies
+
+The allocation strategies allow defining the rule that is used during the stock
+allocation. At this moment two possible options are available:
+
+- Prioritize warehouses by sorting order (`PRIORITIZE_SORTING_ORDER`) - allocate stocks
+  according to the warehouses' order within the channel. If stock is insufficient,
+  the remaining quantity is allocated in the next warehouse on the list
+  and repeated if necessary.
+  To change the warehouses' order within the channel, use `channelReorderWarehouses` mutation.
+- Prioritize warehouses with highest stock (`PRIORITIZE_HIGH_STOCK`) - allocate stock
+  in a warehouse with the most stock. If not enough stock is available in a single
+  warehouse, the remaining quantity is allocated in the next warehouse on the list
+  and repeated if necessary.
```

Comment:
Perhaps continue is less ambiguous, compared to repeated
`warehouse on the list and continues to the next one if necessary.`